control-plane/deploy.sh

```
@@ -0,0 +1,50 @@
+# Requires GRAFANA_ADMIN_PASSWORD from environment or /etc/ebpf-vpc/control-plane-config.env
+
+CONFIG_FILE="/etc/ebpf-vpc/control-plane-config.env"
+NAMESPACE="ebpf-vpc-control"
+SECRET_NAME="observability-grafana-creds"
+
+echo "Starting Control Plane Deployment..."
+
+# 1. Load Configuration (if present)
+if sudo test -f "$CONFIG_FILE"; then
+  set -a
+  source <(sudo cat "$CONFIG_FILE")
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)